Prose/ProseLib/Sources/ProseCoreTCA/Types/Message.swift

```diff
@@ -93,14 +94,16 @@ extension MessageKind {
 }
 
 public struct MessageReactions: Equatable {
-  var reactions: [Reaction: Set<JID>]
+  public typealias WrappedValue = OrderedDictionary<Reaction, OrderedSet<JID>>
```

Comment:
Why do the JIDs need to be ordered as well?

Reply:
You're right, there are no reason right now because we're not displaying the names, but as it's chronologically ordered I thought it'd make sense to keep the order.

Reply:
So should I use `Set` or should I add a doc comment?

Reply:
Please make it as simple as possible and use a Set.